src/script/notification/NotificationRepository.test.ts

```diff
@@ -542,8 +553,14 @@ describe('NotificationRepository', () => {
 
   describe('shows a well-formed group notification', () => {
     beforeEach(() => {
-      const titleLength = NotificationRepository.CONFIG.TITLE_LENGTH;
-      const titleText = `${message.user().name()} in ${conversation.display_name()}`;
+      const titleLength = NotificationRepository.CONFIG.TITLE_MAX_LENGTH;
+      const titleSectionLength = NotificationRepository.CONFIG.TITLE_LENGTH;
+
+      const titleText = `${message.user().name()} in ${truncate(
+        conversation.display_name(),
+        titleSectionLength,
+        false,
+      )}`;
```

Comment:
This piece of code is duplicated multiple times. If possible let's refactor it to a single function